builds/android/src/org/easyrpg/player/SettingsActivity.java

```diff
@@ -285,6 +294,19 @@ public void checkboxVibrateWhenSlidingToAnotherDirection(View v) {
 		editor.commit();
 	}
 
+	//  audio
+	public void checkboxDisableSounds(View v) {
+		CheckBox s = (CheckBox) v; 
+		if(s.isChecked()){
+			editor.putBoolean(getString(R.string.disable_sounds),true);
+		}
+		else{
+			editor.putBoolean(getString(R.string.disable_sounds),false);
+
+		}
+		editor.commit();	
+	}
```

Comment:
Because the if-condition is a bool this can be simplified to a one-liner:
```
editor.putBoolean(getString(R.string.disable_sounds), s.isChecked());
```